packages/commonwealth/server/controllers/server_chains_methods/get_related_communities.ts

```diff
@@ -0,0 +1,43 @@
+import { QueryTypes } from 'sequelize';
+import { sequelize } from '../../database';
+import { ServerChainsController } from '../server_chains_controller';
+
+export type GetRelatedCommunitiesOptions = { base: string };
+
+export type GetRelatedCommunitiesResult = {
+  community: string;
+  icon_url: string;
+  thread_count: number;
+  address_count: number;
```

Comment:
I would add some id/communityId field to be able to identify each record (even though the `community` field should be unique)